src/main/java/com/iexec/sms/secret/web2/Web2SecretRepository.java

```diff
@@ -23,5 +23,9 @@
 import java.util.Optional;
 
 public interface Web2SecretRepository extends CrudRepository<Web2Secret, String> {
-    Optional<Web2Secret> findByOwnerAddressAndAddressAllIgnoreCase(String ownerAddress, String address);
+    Optional<Web2Secret> findByHeader(Web2SecretHeader header);
+
+    default Optional<Web2Secret> find(String ownerAddress, String address) {
+        return findByHeader(new Web2SecretHeader(ownerAddress, address));
```

Comment:
```suggestion
        return findById(new Web2SecretHeader(ownerAddress, address));
```
The interface should be  `public interface Web2SecretRepository extends CrudRepository<Web2Secret, Web2SecretHeader>`

Reply:
Good point! Please see https://github.com/iExecBlockchainComputing/iexec-sms/pull/140/commits/9e0e25ddf1284532e8dffd3da038f5333a85a186.